traefik/crds/ingressroute.yaml

```diff
@@ -1,12 +1,196 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
```

Comment:
```suggestion

Do we need that line?

Reply:
Its autogenerated by kubebuilder, I kept it in in case someone ran a diff against the published CRD references:

https://github.com/traefik/traefik/tree/master/docs/content/reference/dynamic-configuration

Reply:
Great! Thank you for explaining that.